### PR TITLE
changes to prompt structure

### PR DESCRIPTION
**Pull request by reeceshuttle**
Just ignore the name of this branch. It doesn't read the cache files. I've tweaked the prompt format so that a colon is added after the Q and A's and ensured that a tiny bit of structure is allowed for the examples inside of the minify_pddl_problem() function. This will be minor, since it will be two ':' tokens and 4 '\n' tokens for each example.

### CODE REVIEW

File: llm4pddl/utils.py [hunk 1]
```diff
@@ -220,6 +220,15 @@ def minify_pddl_problem(problem: str) -> str:
         partially_flattened = new_problem.replace(')\n(', ')(')
         # Removing new lines in init, a space is needed between
         new_problem = partially_flattened.replace('\n', ' ')
+        # Add new lines for the question and answer tokens
+        new_problem = new_problem.replace('Q: ', 'Q:')
+        new_problem = new_problem.replace('A: ', 'A:')
```
Comment (reeceshuttle) [inline llm4pddl/utils.py:225]: should I use `utils.LLM_ANSWER_TOKEN` instead of `A` here? and same with Q?

Comment (tomsilver) [reply llm4pddl/utils.py:225]: yes please!

**Comment by reeceshuttle** [comment]
I didnt notice that the answer and question tokens already had ':', so I removed my changes to those parts.

**Review by tomsilver — APPROVED** [review]
Good catch! LGTM now

**Pull request merged**